lib/exabgp/application/healthcheck.py

```diff
@@ -456,14 +448,15 @@ def loop(options):
         "FALLING",              # Checks are currently failing.
         "UP",                   # Service is considered as up.
         "DOWN",                 # Service is considered as down.
+        "EXIT",                 # Exit state
     )
 
     def exabgp(target):
         """Communicate new state to ExaBGP"""
-        if target not in (states.UP, states.DOWN, states.DISABLED):
+        if target not in (states.UP, states.DOWN, states.DISABLED, states.EXIT):
             return
         # dynamic ip management. When the service fail, remove the loopback
-        if target in (states.DOWN, states.DISABLED) and options.ip_dynamic:
+        if target in (states.DOWN, states.DISABLED, states.EXIT) and options.ip_dynamic:
```

Comment:
I think this is `target == states.EXIT or (target in (states.DOWN, states.DISABLED) and options.ip_dynamic):`

Reply:
My bad, your version is the right one.